src/main/java/org/opentripplanner/api/mapping/StopMapper.java

```diff
@@ -33,7 +33,8 @@ public static ApiStop mapToApi(StopLocation domain, boolean extended) {
     api.code = domain.getCode();
     api.name = domain.getName().toString();
     if (extended) {
-      api.desc = domain.getDescription();
+      String desc = domain.getDescription() != null ? domain.getDescription().toString() : null;
+      api.desc = desc;
```

Comment:
You can use `NonLocalizedString#ofNullable` for these

Reply:
If we are to use `NonLocalizedString#ofNullable` here, we should also change api.desc to be I18String, because this will cause nullpointer in some cases, because we have to call `toString()` here.